libs/wingsdk/src/simulator/simulator.ts

```diff
@@ -865,6 +886,11 @@ export interface ISimulatorResourceInstance {
    * (files, ports, etc).
    */
   cleanup(): Promise<void>;
+
+  /**
+   * Save the resource's state.
```

Comment:
```suggestion
   * Save the resource's state into the state directory.
```

As a convenience, I am wondering if it would be helpful to pass the state directory here so resource implementers won't have to carry it from init...

Reply:
👍